__tests__/cli.test.js

```diff
@@ -18,14 +18,12 @@ describe('cli', () => {
     it('creates a Tailwind config file', () => {
       return cli(['init']).then(() => {
         expect(utils.writeFile.mock.calls[0][0]).toEqual(constants.defaultConfigFile)
-        expect(utils.writeFile.mock.calls[0][1]).toContain('defaultConfig')
```

Comment:
What do you think about checking for `defaultTheme`?

```
expect(utils.writeFile.mock.calls[0][1]).toContain('defaultTheme')
```

Reply:
I'm not sure what I even want to scaffold for the config file when people use `tailwind init` yet so just gonna punt on it entirely for now I think.